Apromore-Custom-Plugins/Account-Portal-Plugin/src/main/resources/META-INF/spring/context.xml

```diff
@@ -44,6 +44,9 @@
     <beans:bean id="portalPlugin2" class="org.apromore.plugin.portal.account.DebugPlugin"/>
     <osgi:service ref="portalPlugin2" interface="org.apromore.plugin.portal.PortalPlugin" auto-export="interfaces" />
     -->
+    <osgi:service interface="org.apromore.plugin.portal.PortalPlugin" auto-export="interfaces">
```

Comment:
You can remove the `auto-export` attribute since you're explicitly defining the exported interface here.